vocab/endpoints.yaml

```diff
@@ -1,6 +1,6 @@
 $schema: ../codegen/schema.yaml
 name: Endpoints
-uri: "https://www.w3.org/ns/activitystreams#Endpoints"
+uri: "https://www.w3.org/ns/activitystreams#endpoints"
```

Comment:
> When I look around vocabulary definitions (yaml files), `.url` property seems valid url with anchor. But the `.url` property in `endpoints.yaml` wasn't valid because there is no `Endpoints` anchor. There was `endpoints` instead (lowercase).

Yes, `Endpoints` class actually is not a regular class defined in Activity Vocabulary.  It purposes to suit Fedify's vocabulary codegen system.  Its IRI (URI) does not matter anyway — so I don't think this change is necessary.

Reply:
Okay, then I'll drop the commit.

Reply:
I dropped the commit and force-pushed. Please review again 🙏🏻